server/reportConfiguration/types/DISORDER.ts

```diff
@@ -639,7 +639,7 @@ const DISORDER: IncidentTypeConfiguration = {
           id: '214747',
           code: 'ORRU ASSISTANCE REQUESTED',
           active: true,
-          label: 'ORRU ASSISTANCE REQUESTED',
+          label: 'Orru assistance requested',
```

Comment:
ORRU is another acronym: Operational Response & Resilience Unit

Reply:
Didn't sport this, I'll add `ORRU` to the preserved words 👍

Reply:
Updated here: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/287/commits/790cdaf0d52e8865428c4d05bd127067f162acdf